EIPS/eip-9000.md

```diff
@@ -0,0 +1,332 @@
+---
+eip: 9000
```

Comment:
```suggestion
eip: 7196
```
Assigning number 7196 for first PR opened #7196 
Please also rename the filename.

Reply:
thanks, changed